extensions/gitlab/package.json

```diff
@@ -62,7 +62,17 @@
       "title": "My Todos",
       "subtitle": "GitLab",
       "description": "My GitLab Todos",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "excludeTodoAuthorUsernames",
+          "type": "textfield",
+          "required": false,
+          "title": "Excclude Todo Authors",
```

Comment:
```suggestion
          "title": "Exclude Todo Authors",
```

Reply:
Can't you add the shared preferences in [the root](https://developers.raycast.com/information/manifest#extension-properties) instead of under a command?

Reply:
ok, preference moved to the root, and the typo was fixed